packages/adobe-sign/src/apis/errors/internal-api-error.ts

```diff
@@ -0,0 +1,9 @@
+import {ApiError} from "./api-error";
+
+export class InternalApiError extends ApiError {
+  statusCode = 500;
+  constructor(message: string){
+    super('Internal API Error - ' + message);
+    Object.setPrototypeOf(this, InternalApiError.prototype);
```

Comment:
If we don't do this, does the prototype end up being ApiError, and we cannot distinguish between an InternalApiError and an AdobeApiError?